viam-cartographer/src/carto_facade/util.cc

```diff
@@ -56,15 +62,21 @@ int read_pcd(std::string pcd, pcl::PCLPointCloud2 &blob) {
                            data_type, data_idx);
     if (res != 0) {
         LOG(ERROR) << "Failed to parse header";
+        return res;
+    } else if (blob.data.size() == 0) {
+        LOG(ERROR) << "Failed to parse header: header has no points";
+        return -1;
     } else {
```

Comment:
Can we remove the enclosing 'else' since the above two blocks return?

Reply:
done